### PR TITLE
Make anchor harmless

### PR DESCRIPTION
**Pull request by zdepodany**
And i had a Prettier (text formatter) on, so rip you and your need for 'Tab = 4x space'

### CODE REVIEW

**Comment by Dormouse759** [comment]
But tab *equals* 4 spaces!
image

File: taborapp/static/css/gallery.css [hunk 1]
```diff
@@ -1,263 +1,267 @@
 .yearAndButton {
-    width: 40%;
-    justify-content: space-evenly;
-    align-items: center;
-    color: #2B2A28;
+  width: 40%;
+  justify-content: space-evenly;
+  align-items: center;
+  color: #2b2a28;
 }
```
Comment (Dormouse759) [inline taborapp/static/css/gallery.css:7]: I strongly disagree, tab == 4 spaces

**Review by Dormouse759 — COMMENTED** [review]
I don't disapprove.

File: taborapp/static/css/gallery.css (hunk 1 above)
Comment (zdepodany) [reply taborapp/static/css/gallery.css:7]: Prettier say NO

**Pull request merged**